activestorage/CHANGELOG.md

```diff
@@ -32,4 +32,18 @@
 
    *George Claghorn*
 
+* Method delegation to attachments now allows `nil` to be returned when the file is not attached.
+
+  Previously, the following would return a `Module::DelegationError` when `file` was not attached, but it now returns `nil`:
+
+  ```ruby
+  class Attachment < ApplicationRecord
+    has_one_attached :file
+
+    delegate :filename, to: :file, allow_nil: true
+  end
+  ```
```

Comment:
A clearer version:

> Methods calls on singular attachments return `nil` when no file is attached.
>
> Previously, assuming the following `User` model, `user.avatar.filename` would raise a `Module::DelegationError` if no avatar was attached:
>
> ```ruby
> class User < ApplicationRecord
>   has_one_attached :avatar
> end
> ```
>
> They now return `nil`.

Also, new changelog entries get added to the top of the file. Can you move yours up?